product_docs/docs/pgd/5/durability/comparing-commit-scopes.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Comparing Commit Scope Kinds
+navTitle: Comparing Commit Scope Kinds
+deepToC: true
+---
+
+## Comparing Group Commit and Synchronous Commit
+
+### How Group Commit works
+
+A Group Commit first writes a prepared transaction to local disk and waits till it is replicated to enough nodes and acknowledged by them before then committing it locally.
+
+This also gives resiliency to failure of N nodes, where N is the number of nodes required to acknowledge the transaction. 
+Commit scope syntax allows you to specify N as part of the rule.
+
+### How Synchronous Commit works
+
+A Synchronous Commit first commits to the local disk and then waits till the transaction is replicated to N other nodes, as per rules of the commit scope. 
+On receiving confirmation that the transaction has been received, applied or made durable, again based on commit scope settings, the client gets a completion acknowledgement of the transaction.
+
+Synchronous commit increases resiliency - it ensures that the transaction is received by more than N nodes before it is acknowledged to caller. 
+This makes it resilient to failure of N nodes.
+When conflicts occur, with Synchronous Commit the conflict resolution is always the default async PGD conflict resolution.
+
+### Network partitions and node crashes
+
+Consider a network partition where the originating node is on one side of the partition and the replicas on the other.
+We will look at the state of each kind, before and after connectivity is restored.
+
+#### Group Commit and network partitions
+
+With Group Commit and before restoration, the origin node will have prepared a transaction and wait for confirmations which will not arrive.
+It could timeout and abort the transaction or just wait if the timeout setting is long enough.
+Writes that are recieved by the replicas will be held back if the nodes have prepared transactions from the now-disconnected origin. If there are no prepared transactions, they can continue to recieve writes.
```

Comment:
if the nodes have prepared transactions from the now-disconnected origin and they hold conflicting locks.  If there are no conflicts, they can continue to receive writes.